src/map/status.cpp

```diff
@@ -8804,6 +8804,7 @@ t_tick status_get_sc_def(struct block_list *src, struct block_list *bl, enum sc_
 #endif
 			break;
 		case SC_STONE:
```

Comment:
I think we need to remove SC_STONE here because otherwise you can resist the transition from SC_STONEWAIT to SC_STONE.

Alternatively you could also make the transition itself unavoidable.